internal/csi-addons/rbd/replication_test.go

```diff
@@ -438,58 +440,118 @@ func TestCheckRemoteSiteStatus(t *testing.T) {
-			`replaying,{"bytes_per_second":0.0,"bytes_per_snapshot":149504.0,"local_snapshot_timestamp":1662655501,"remote_snapshot_timestamp":1662655501}`,
-			timestamppb.New(time.Unix(1662655501, 0)),
-			"",
+			description: `replaying, {"bytes_per_second":0.0,"bytes_per_snapshot":81920.0,"last_snapshot_bytes":81920,"last_snapshot_sync_seconds":56743,"local_snapshot_timestamp":1684675261,"remote_snapshot_timestamp":1684675261,"replay_state":"idle"}`,
+			info: localstruct{
+				lastsynctime:     timestamppb.New(time.Unix(1684675261, 0)),
+				lastsyncduration: durationpb.New(duration),
+				lastsyncbytes:    81920,
+			},
+			expectedErr: "",
 		},
 		{
-			"empty description",
-			"",
-			nil,
-			corerbd.ErrLastSyncTimeNotFound.Error(),
+			name:        "empty description",
+			description: "",
+			info: localstruct{
+				lastsynctime:     nil,
+				lastsyncduration: nil,
+				lastsyncbytes:    0,
+			},
+			expectedErr: corerbd.ErrLastSyncTimeNotFound.Error(),
 		},
 		{
-			"description without local_snapshot_timestamp",
-			`replaying,{"bytes_per_second":0.0,"bytes_per_snapshot":149504.0,"remote_snapshot_timestamp":1662655501}`,
-			nil,
-			"",
+			name: "description without last_snapshot_bytes",
+			//nolint:lll // sample output cannot be split into multiple lines.
+			description: `replaying, {"bytes_per_second":0.0,"last_snapshot_sync_seconds":56743,"local_snapshot_timestamp":1684675261,"remote_snapshot_timestamp":1684675261,"replay_state":"idle"}`,
+			info: localstruct{
+				lastsyncduration: durationpb.New(duration),
+				lastsynctime:     timestamppb.New(time.Unix(1684675261, 0)),
+				lastsyncbytes:    0,
+			},
+			expectedErr: "",
 		},
 		{
-			"description with invalid JSON",
-			`replaying,{"bytes_per_second":0.0,"bytes_per_snapshot":149504.0","remote_snapshot_timestamp":1662655501`,
-			nil,
-			"failed to unmarshal",
+			name: "description without local_snapshot_time",
+			//nolint:lll // sample output cannot be split into multiple lines.
+			description: `replaying, {"bytes_per_second":0.0,"bytes_per_snapshot":81920.0,"last_snapshot_bytes":81920,"last_snapshot_sync_seconds":56743,"remote_snapshot_timestamp":1684675261,"replay_state":"idle"}`,
+			info: localstruct{
+				lastsyncduration: nil,
+				lastsynctime:     nil,
+				lastsyncbytes:    0,
+			},
+			expectedErr: corerbd.ErrLastSyncTimeNotFound.Error(),
 		},
 		{
-			"description with no JSON",
-			`replaying`,
-			nil,
-			corerbd.ErrLastSyncTimeNotFound.Error(),
+			name: "description without last_snapshot_sync_seconds",
+			//nolint:lll // sample output cannot be split into multiple lines.
+			description: `replaying, {"bytes_per_second":0.0,"bytes_per_snapshot":81920.0,"last_snapshot_bytes":81920,"local_snapshot_timestamp":1684675261,"remote_snapshot_timestamp":1684675261,"replay_state":"idle"}`,
+			info: localstruct{
+				lastsyncduration: nil,
+				lastsynctime:     timestamppb.New(time.Unix(1684675261, 0)),
```

Comment:
This should be nil as your comment in 
replicaiton.go line 779.

```
	// In case there is no last snapshot sync seconds, it returns nil as the
	// LastSyncDuration is optional.
```

Reply:
`lastsyncduration: nil,` yes its nil only right?